packages/kamino-sdk/src/services/JupService.ts

```diff
@@ -0,0 +1,35 @@
+import { Connection, PublicKey } from '@solana/web3.js';
+import { SolanaCluster } from '@hubbleprotocol/hubble-config';
+import axios from 'axios';
+
+export type SwapTransactionsResponse = {
+  setupTransaction: string | undefined;
+  swapTransaction: string;
+  cleanupTransaction: string | undefined;
+};
+
+export class JupService {
+  private readonly _connection: Connection;
+  private readonly _cluster: SolanaCluster;
+
+  constructor(connection: Connection, cluster: SolanaCluster) {
```

Comment:
Are connection and cluster even used here?

Reply:
not yet, but they will be in the next PR